src/main/java/bio/terra/service/snapshotbuilder/utils/constants/PersonConstants.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.service.snapshotbuilder.utils.constants;
+
+public final class PersonConstants {
+  private PersonConstants() { }
+  public static final String PERSON = "person";
+  public static final String PERSON_ID = "person_id";
```

Comment:
This could be in either `PersonConstants` or `ConditionOccurrenceConstants`? This value is present in both tables.

Reply:
I'm not sure, what are your thoughts on putting it in both? I don't know if I even like that idea